src/stdio_mgr/stdio_mgr.py

```diff
@@ -130,10 +132,22 @@ def append(self, text):
         """
         pos = self.tell()
         self.seek(0, self.SEEK_END)
-        retval = self.write(text)
+        retval = self._buf.write(text.encode(self.encoding))
```

Comment:
~Is it better here to go through the internal reference `self._buf` rather than the wrapper's `self.buffer` for some particular reason?~

~Or, does it pretty much not matter?~

I think I get it, per the above: `self._buf` is the R/W `BytesIO`, whereas `self.buffer` is the read-only `BufferedReader`?

Reply:
yup.  I could possibly merge `._buf` and `.buffer`, but 

a) then the r/w nature of ._buf is exposed in .buffer and then it is less like the real `sys.stdin`,

b) it would be funky games with attrs , or removal of attrs.

Reply:
Nah, I think I get it, definitely prefer leaving it with `._buf` and `.buffer`.